clickable.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Can we move this to the telegram subdirectory ?

Reply:
I'll check it.

Reply:
Cannot move this in the telescope subdirectory. It needs to be on the same level as the subdirs project.